### ISSUE TITLE
Bugs in Version 0.20.0

### ISSUE THREAD

**Issue by mohsinanayat** [TeamNewPipe/NewPipe#4396]
There are so many bugs in recent version like loading circles, error while loading videos

### Actual behaviour
While searching videos it shows error i,e sorry that should not have happened ,when we try to watch video. Also Loading circle shows twice..(As shown in screenshots)

### Expected behavior
Serious bug present in Code

### Screenshots/Screen recordings

![Screenshot_2020-10-05-08-34-55-21](https://user-images.githubusercontent.com/63046281/95036882-e1794d00-06e6-11eb-8745-21a36a6af5c1.jpg)
![Screenshot_2020-10-05-08-34-30-08](https://user-images.githubusercontent.com/63046281/95036914-f0f89600-06e6-11eb-95e7-1cd7deaa7e5a.jpg)

## Exception
* __User Action:__ requested comments
* __Request:__ https://m.youtube.com/watch?v=FUsemuP5fJ8
* __Content Country:__ US
* __Content Language:__ en-US
* __App Language:__ en_US
* __Service:__ YouTube
* __Version:__ 0.20.0
* __OS:__ Linux realme/RMX2002/RMX2002L1:10/QP1A.190711.020/1594211000:user/release-keys 10 - 29
<details><summary><b>Crash log </b></summary><p>

```
java.lang.StringIndexOutOfBoundsException: String index out of range: -15
	at java.lang.String.substring(String.java:2064)
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeCommentsExtractor.findValue(YoutubeCommentsExtractor.java:165)
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeCommentsExtractor.getInitialPage(YoutubeCommentsExtractor.java:51)
	at org.schabi.newpipe.extractor.utils.ExtractorHelper.getItemsPageOrLogError(ExtractorHelper.java:19)
	at org.schabi.newpipe.extractor.comments.CommentsInfo.getInfo(CommentsInfo.java:40)
	at org.schabi.newpipe.extractor.comments.CommentsInfo.getInfo(CommentsInfo.java:25)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getCommentsInfo$7(ExtractorHelper.java:155)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$Xyj84g1UeNZqX9LVYNvWZsdT-kI.call(Unknown Source:4)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.Flowable.subscribe(Flowable.java:14935)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)

```
</details>
<hr>

**Comment by opusforlife2** [comment]
Why did you delete the checkboxes? It clearly says one bug report per issue. And did you check for duplicates?

**Comment by mohsinanayat** [comment]
I don't care

**Comment by mohsinanayat** [comment]
You closed it,it is ur problem not mine, so keep this in u r mind

**Comment by opusforlife2** [comment]
It's alright. I won't lose any sleep over this, I assure you. Don't worry about me. 🤭

**Comment by mohsinanayat** [comment]
You should welcome everyone,who takes his precision time to report issue , it takes 10 mins to report issue but it takes Seconds to close it...

**Comment by opusforlife2** [comment]
I know. I spend a lot of time when I open issues as well. But if you could take an extra 5 minutes to do the checkboxes, it makes everyone's life easier. You already have the content. You just need to copy paste it into separate bug reports.

**Comment by mohsinanayat** [comment]
> I know. I spend a lot of time when I open issues as well. But if you could take an extra 5 minutes to do the checkboxes, it makes everyone's life easier. You already have the content. You just need to copy paste it into separate bug reports.
> 
> Edit: but make sure your issues aren't duplicates, because they'll be closed again and you'll be even more irritated. I know that feeling, too.

It would have better if u would have warn. me,i would have change it as soon as possible my dear bro....

**Comment by opusforlife2** [comment]
I can't know in advance that you're going to open an issue, now, can I? Sadly, it is only after I read a new issue that I can point out such problems. Personally, I have accepted that this sort of minor friction is an inevitable part of maintainership.

**Comment by mohsinanayat** [comment]
> I can't know in advance that you're going to open an issue, now, can I? Sadly, it is only after I read a new issue that I can point out such problems. Personally, I have accepted that this sort of minor friction is an inevitable part of maintainership.

Remember there are still unlimited number of bugs present in this app, that i saw but can't report becoz.i know u will not solve it/close it...

**Comment by opusforlife2** [comment]
> .i know u will not solve it/close it...

Solving/even deciding to solve, is up to developers, not us. And I only close issues if they are duplicates or not maintainable. :3

**Comment by mohsinanayat** [comment]
Ok dear,can u tell me where are u from?

**Comment by opusforlife2** [comment]
India. ;)

**Comment by mohsinanayat** [comment]
From which State?

**Comment by opusforlife2** [comment]
Take a guess: It's the most polluted place on Earth.

**Comment by mohsinanayat** [comment]
I think Delhi....

**Comment by mohsinanayat** [comment]
Am i right????

**Comment by opusforlife2** [comment]
xD Yeah.

**Comment by mohsinanayat** [comment]
Also we are from J&K where - govt.says internet is not mandatory 😂 and it is not a right......

**Comment by opusforlife2** [comment]
Alright. We've had fun, now I'm marking all our messages as off-topic, okay? Come on IRC if you want to chat generally. 👍

**Comment by mohsinanayat** [comment]
OK Dear,Can give me your WhatsApp number so that we can chat privately.

**Comment by opusforlife2** [comment]
Sorry, no. Here I exist as my Github username only. I am not a real person. ¯\\\_(ツ)\_/¯

Also, don't use Whatsapp. It is owned by Facebook. O_o

**Comment by mohsinanayat** [comment]
So then how can i chat with you privately?

**Comment by mohsinanayat** [comment]
> Sorry, no. Here I exist as my Github username only. I am not a real person. ¯\\\_(ツ)\_/¯
> 
> Also, don't use Whatsapp. It is owned by Facebook. O_o

You are not a real person,what does that mean🤔.....

**Comment by opusforlife2** [comment]
🤫

**Comment by mohsinanayat** [comment]
> 🤫 

Ur choice 😏
